base/services/svchost/globals.c

```diff
@@ -234,7 +234,7 @@ ScDomainIdToSid (
 NTSTATUS
 NTAPI
 ScAllocateAndInitializeSid (
-    _Out_ PVOID *Sid,
+    _Out_ PSID *Sid,
```

Comment:
```suggestion
    _Out_ PSID* Sid,
```
(keeping "consistency" with the adjustments done in the structures below)

Reply:
no.